projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -23,7 +23,18 @@ export class IgxMaskDirective implements OnInit, AfterViewChecked, ControlValueA
      * ```
      */
     @Input('igxMask')
-    public mask: string;
+    public get mask(): string {
+        return this._mask;
+    }
+
+    public set mask(val: string) {
+        // B.P. 9th June 2021 #7490
+        this._oldMask = this._mask || this.defaultMask;
```

Comment:
having `_mask` we may remove `_oldMask` and `defaultValue` so this line becomes needless.